src/modules/commentsApi.js

```diff
@@ -0,0 +1,23 @@
+export const addComments = async (itemId, userName, userComment) => {
+  const addComment = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/3vnHMFG69E4WPq0bdrAl/comments', {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: itemId,
+      username: userName,
+      comment: userComment,
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  const commentText = await addComment.text();
+  return commentText;
+};
```

Comment:
- [ ] You are repeating yourself too much. This return can be 👇🏾 

```javascript
// const commentText = await addComment.text();
  // return commentText;
  return await addComment.text();
```

I have seen this in many places. Kindly refactor your code and try to avoid unnecessary code.